skbio/io/format/stockholm.py

```diff
@@ -0,0 +1,415 @@
+"""
+Stockholm format (:mod:`skbio.io.format.stockholm`)
+===================================================
+
+.. currentmodule:: skbio.io.format.stockholm
+
+The Stockholm format is a multiple sequence alignment format written in Markup
+as opposed to a simple text-based format. Data is stored on seperate lines,
+each with a unique data 'name' preceding it. The alignment can just contain
+data, or contain data and related metadata.
+
+An example Stockholm file, taken from [1]_:
+
+.. code-block:: none
+
+    # STOCKHOLM 1.0
+    #=GF ID    UPSK
+    #=GF SE    Predicted; Infernal
+    #=GF SS    Published; PMID 9223489
+    #=GF RN    [1]
+    #=GF RM    9223489
+    #=GF RT    The role of the pseudoknot at the 3' end of turnip yellow mosaic
+    #=GF RT    virus RNA in minus-strand synthesis by the viral RNA-dependent \
+RNA
+    #=GF RT    polymerase.
+    #=GF RA    Deiman BA, Kortlever RM, Pleij CW;
+    #=GF RL    J Virol 1997;71:5990-5996.
+    AF035635.1/619-641             UGAGUUCUCGAUCUCUAAAAUCG
+    M24804.1/82-104                UGAGUUCUCUAUCUCUAAAAUCG
+    J04373.1/6212-6234             UAAGUUCUCGAUCUUUAAAAUCG
+    M24803.1/1-23                  UAAGUUCUCGAUCUCUAAAAUCG
+    #=GC SS_cons                   .AAA....<<<<aaa....>>>>
+    //
+
+Format Support
+==============
+**Has Sniffer: Yes**
+
+**State: Experimental as of 0.4.0-dev.**
+
++------+------+---------------------------------------------------------------+
+|Reader|Writer|                          Object Class                         |
++======+======+===============================================================+
+|Yes   |No    |:mod:`skbio.alignment.TabularMSA`                              |
++------+------+---------------------------------------------------------------+
+
+Format Specification
+====================
+Stockholm format contains two types of data: raw DNA, RNA, or Protein data and
```

Comment:
This is a little confusing the way it's wording, saying there are two types of data then listing 3. Maybe reword it to something like: "Stockholm format contains two types of data: nucleotides (DNA or RNA) or Protein...". Or some other rearrangement.